controlpanel/frontend/views/user.py

```diff
@@ -36,11 +51,20 @@ def get_context_data(self, **kwargs):
                 continue
 
             try:
-                last_login = dateutil.parser.parse(user.get('last_login'))
+                auth0_last_login = user.get("last_login")
+                if auth0_last_login:
+                    last_login = dateutil.parser.parse(user.get('last_login'))
+                else: 
+                    db_user = auth0_mapper.get(auth0_id, None)
+                    if db_user:
+                        last_login = db_user.last_login
             except (TypeError, ValueError):
                 last_login = None
 
-            last_logins.append((auth0_id, last_login))
+            last_logins.append((auth0_id, {
+                "last_login": last_login,
+                "unused": last_login.utcnow() < three_months_ago(),
```

Comment:
Maybe this is a bit objective but when I saw `unused` in the templates it wasn't immediately clear what this was (when I just started to look at this) so maybe call this variables `not_logged_recently` may help (or some better name than this).

Reply:
I really don't mind what we call it, ;-) but I chose `unused` because it's short and simple and is a good flag name to indicate the account maybe no longer used. The addition of the comments in the code should clarify what it's supposed to represent.